docker-compose.prod.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  frontend:
```

Comment:
I changed the name to avoid errors when you switch between the dev/prod versions.
In this way, will throw a known error, to remember that it's needed to `docker-compose down` before the version switch.